src/Mollie/WC/Helper/PaymentMethodsIconUrl.php

```diff
@@ -24,7 +24,7 @@ public function __construct(array $paymentMethodImages)
      */
     public function svgUrlForPaymentMethod($paymentMethodName)
     {
-        return $this->paymentMethodImages? $this->paymentMethodImages[$paymentMethodName]->svg : $this->fallToAssets($paymentMethodName);
+        return isset($this->paymentMethodImages[$paymentMethodName]->svg)? $this->paymentMethodImages[$paymentMethodName]->svg : $this->fallToAssets($paymentMethodName);
```

Comment:
There should be a space before `?` 

If you need we can have a look together about the configuration for coding standard in you IDE